modules/proxy-namespace-procedure.adoc

```diff
@@ -0,0 +1,42 @@
+
+[[red-hat-quay-proxy-namespace-procedure]]
+= Using {productname} to proxy a namespace in a registry
+
+The following procedure describes how you can use {productname} to proxy a namespace in a registry. This procedure is set up to proxy quay.io, which allows users to use `podman` to pull any public image from any namespace on quay.io.
+
+.Prerequisites
+
+*
```

Comment:
Should we mention here that the [FEATURE_PROXY_CACHE](https://github.com/quay/quay/blob/master/config.py#L47) config needs to set to `true` in the `config.yaml` for this feature to become available?
Even our own QE was confused by that when I failed to mention this config in the testing steps I gave them 😅